src/mame/drivers/ti85.cpp

```diff
@@ -1032,6 +1032,13 @@ ROM_START (ti84pv2)
 	ROMX_LOAD( "ti84pv2v255mp.bin", 0x00000, 0x100000, CRC(3f8574a6) SHA1(0f88e719512f2691fff6c8bcc89292158086f841), ROM_BIOS(8) )
 ROM_END
 
+ROM_START (ti84pv2)
```

Comment:
This machine name is already used.  Looking at your COMP( addition, it looks like it should be named ti84pov2?